src/SurfaceFluxes.jl

```diff
@@ -76,6 +82,7 @@ struct SurfaceFluxConditions{FT <: Real}
     Cd::FT
     Ch::FT
     evaporation::FT
+    is_converged::Bool
```

Comment:
We're trying to reduce the number of fields in `SurfaceFluxConditions`, why do we need an entire 2D spherical shell of Bools indicating whether we're converging or not?

Reply:
Fixing in #57; This was included for cases where `error_on_non_convergence == false` wouldn't interrupt the solution; but you're right; it is unnecessary to carry this for every timestep in cases set up to throw errors on non-convergence.